system-event-models/src/main/java/com/telecominfraproject/wlan/systemevent/aggregation/models/CustomerPortalDashboardPartialEvent.java

```diff
@@ -43,6 +43,8 @@
 	private AtomicLong trafficBytesUpstream = new AtomicLong();
 	
 	private Map<String, AtomicInteger> clientCountPerOui = new ConcurrentHashMap<>();
+	
+	private Map<String, AtomicInteger> alarmsCountBySeverity = new ConcurrentHashMap<>();
```

Comment:
Why using string, when the severity is defined as StatusCode?

Reply:
Resolved in the latest commit. 

Moved CustomerPortalDashboardPartialEvent to status-models to access StatusCode and avoid dependency issues